lsp-mode.el

```diff
@@ -7192,19 +7192,32 @@ nil."
 
 
 ;; Download URL handling
-(cl-defun lsp-download-install (callback error-callback &key url store-path &allow-other-keys)
-  (let ((url (lsp-resolve-value url))
-        (store-path (lsp-resolve-value store-path)))
+(cl-defun lsp-download-install (callback error-callback &key url store-path decompress &allow-other-keys)
```

Comment:
It is Elisp convention to use keywords if you have a fixed set of options. I.e. `:gzip`, `:zip`. Can you do this change without force-pushing?

Reply:
TIL about keyword symbols. Pushed the commit there, hope I'm understanding this correctly